### DISCUSSION TITLE
Request parameters allure-maven:serve

### DISCUSSION THREAD

**Discussion by svs-sp** [allure-framework/allure2#1808] [Questions & Support]
I am running the server with
mvn io.qameta.allure:allure-maven:serve
command how can I add a host and port parameters to the request?

**Comment by baev** [comment]
allure-maven plugin has `serveHost` and `servePort` configuration parameters. Besides that, you can use `allure.serve.host` and `allure.serve.port` system properties if you want to override this values via command options: 

```bash
mvn io.qameta.allure:allure-maven:serve -Dallure.serve.host="http://example.org" -Dallure.serve.port=1234
```

**Comment by svs-sp** [comment]
Thank you very much!
